### DISCUSSION TITLE
Press charges

### DISCUSSION THREAD

**Discussion by martinlopez799** [community/community#103054] [New to GitHub]
Hi I'm new to these and I stumbled upon by finding out there using the development program on me I'm really sorry but frustrated how he's taking data from 3 of my  2tablets and then my mom's account

**Comment by martinlopez799** [comment]
I'm a Flintstone to this

**Comment by ghostinhershell** [comment]
Hi @martinlopez799, 👋🏾 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
